mtp_api/apps/credit/urls.py

```diff
@@ -9,5 +9,5 @@
     url(r'^credits/actions/lock/$', views.LockCredits.as_view(), name='credit-lock'),
     url(r'^credits/actions/unlock/$', views.UnlockCredits.as_view(), name='credit-unlock'),
     url(r'^credits/senders/$', csrf_exempt(views.SenderList.as_view()), name='sender-list'),
-    url(r'^credits/recipients/$', csrf_exempt(views.RecipientList.as_view()), name='recipient-list'),
+    url(r'^credits/prisoners/$', csrf_exempt(views.PrisonerList.as_view()), name='prisoner-list'),
```

Comment:
There was a reason for calling it 'recipients' rather than 'prisoners' which is that they are not equal sets - although all recipients are prisoners, not all prisoners are recipients. Doesn't really matter, though.